aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -384,6 +384,9 @@ webBuilder.ConfigureKestrel(serverOptions =>
 });
 ```
 
+> [!NOTE]
+> Endpoints created by calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.Listen*> **before** calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.ConfigureEndpointDefaults*> will not have the defaults applied.
```

Comment:
Super nit:

```suggestion
> Endpoints created by calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.Listen*> **before** calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.ConfigureEndpointDefaults> will not have the defaults applied.
```

There's only one ConfigureEndpointDefaults method. I doesn't look like the `*`s are showing up in the final render anyway.

Reply:
Leave the `*` (required) and add the `s` in there.